onelogin_aws_cli/cli.py

```diff
@@ -48,6 +50,22 @@ def login(args=sys.argv[1:]):
     api.save_credentials()
 
     if args.renewSeconds:
-        while True:
-            time.sleep(args.renewSeconds)
+
+        running = Event()
+
+        def _interrupt_handler(signal_num: int, *args):
+            running.set()
+            print("Shutting down Credentials refresh process...")
+
+        # Handle sigterms
+        # This must be done here, as signals can't be caught down the stack
+        for sig_type in list(SignalRepr):
+            signal.signal(sig_type.value, _interrupt_handler)
+
+        # @TODO We should check if the credentials are going to expire
+        # in the immediate future, rather than constantly hitting
+        # the AWS API.
+        running.clear()
+        while not running.is_set():
             api.save_credentials()
+            running.wait(args.renewSeconds)
```

Comment:
This is all backwards to the example in https://github.com/physera/onelogin-aws-cli/pull/48#pullrequestreview-103709968 because `wait()` does not work while `is_set()` returns true.